### PR TITLE
[bitnami/cassandra] add hostPort-support

### PR DESCRIPTION
**Pull request by stefan-work**
### Description of the change

Expose ports of the cassandra-containers as k8s hostPort.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/cassandra/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: cassandra
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/cassandra
   - http://cassandra.apache.org
-version: 9.6.1
+version: 9.7.1
```
Comment (carrodher) [inline bitnami/cassandra/Chart.yaml:26]: Thanks for the PR, please note there is a typo when bumping the version
```suggestion
version: 9.7.0
```

Comment (stefan-work) [reply bitnami/cassandra/Chart.yaml]: Ouch... fixed in 39e5ea4

File: bitnami/cassandra/values.yaml [hunk 2]
```diff
@@ -770,6 +783,15 @@ metrics:
     http: 8080
     jmx: 5555
 
+  ## Cassandra ports to be exposed as hostPort
```
Comment (carrodher) [inline bitnami/cassandra/values.yaml:786]:
```suggestion
  ## Metrics ports to be exposed as hostPort
```

Comment (carrodher) [reply bitnami/cassandra/values.yaml:786]: Just a minor comment, everything else LGTM

Comment (stefan-work) [reply bitnami/cassandra/values.yaml]: Fixed in fac1935

**Review by carrodher — APPROVED** [review]

**Pull request merged**